src/internal/components/option/index.tsx

```diff
@@ -58,33 +58,33 @@ const Option = ({
   );
 
   return (
-    <div
+    <span
```

Comment:
Does this cause any visual changes since these will now be in inline instead of block elements?

Reply:
Good question. All of the elements I changed had `display: flex `already so there was no change to the computed display value. Additionally, visual regression passed with no issues.